sdk/signalr/azure-messaging-webpubsubservice/examples/send_messages.py

```diff
@@ -0,0 +1,67 @@
+import io
+import logging
+import os
+
+from azure.core.credentials import AzureKeyCredential
+from azure.messaging.webpubsubservice import WebPubSubServiceClient
+from azure.messaging.webpubsubservice.rest import *
+
+endpoint = os.environ['ENDPOINT']
+api_key = os.environ['API_KEY']
+
+LOG = logging.basicConfig(level=logging.DEBUG)
+
+client = WebPubSubServiceClient(endpoint, credential=AzureKeyCredential(api_key), tracing_enabled=True)
+
+# Send message to everybody on the given hub...
+request = build_send_to_all_request('ahub', json={ 'Hello': 'all!' })
```

Comment:
If I pass in a string to the `json` parameter - will that just become the payload?